packages/manager-components/src/components/clipboard/clipboard.component.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { OsdsClipboard, OsdsText } from '@ovhcloud/ods-components/react';
+import { useTranslation } from 'react-i18next';
+import './translations';
+
+export interface ClipboardProps {
+  value: string;
+}
```

Comment:
Use the Attribute type of OsdsClipboard instead and pass all the props
There an `inline` property for example that we can use